Translation/en/RedirectedResources/assets/abdata/studio/info/01/map_01/translation.txt

```diff
@@ -2,33 +2,33 @@
 // Dumped for Koikatsu Sunshine v1.0.3 (KoikatsuSunshine) by KKS_TextDump v1.4.5.2
 //
 //名称=
```

Comment:
Other type of name indicator, maybe it was a placeholder from previews versions?